man/man5/zfs-module-parameters.5

```diff
@@ -1752,14 +1769,97 @@ Default value: \fB2\fR.
 .ad
 .RS 12n
 Max concurrent I/Os per top-level vdev (mirrors or raidz arrays) allowed during
-scrub or resilver operations.
+legacy scrub or resilver operations.
 .sp
 Default value: \fB32\fR.
 .RE
 
 .sp
 .ne 2
 .na
+\fBzfs_scan_md_maxinflight\fR (int)
+.ad
+.RS 12n
+Max concurrent I/Os allowed during scrub or resilver metadata scanning phase.
+.sp
+Default value: \fB1024\fR.
```

Comment:
My only problem with using `zfs_top_maxinflight` here is that its really not accurate. The current code just assumes that all blocks are being distributed evenly to all top-level vdevs. So we just end up setting a global limit of `zfs_top_maxinflight * <number of top-level vdevs>`. I'm ok keeping that logic if that's what we want. Maybe the name just needs to be changed to be a bit less misleading about what its doing.

As far as the value, I'll lower it so that we end up with the same ~500ms maximum. Again, sorry. I forgot that not everyone runs zfs on large 100+TB servers. :)